nimble/host/src/ble_hs_flow.c

```diff
@@ -231,7 +231,10 @@ ble_hs_flow_startup(void)
 
     /* Assume failure. */
     ble_hci_trans_set_acl_free_cb(NULL, NULL);
+
+    #if MYNEWT_VAL(SELFTEST)
     ble_npl_callout_stop(&ble_hs_flow_timer);
+    #endif
```

Comment:
ah sorry, indentation is still not fixed in the code.

Reply:
@rymanluk Done!! :neutral_face: